clouddriver-cloudfoundry/src/main/java/com/netflix/spinnaker/clouddriver/cloudfoundry/artifacts/CloudFoundryArtifactCredentials.java

```diff
@@ -29,18 +29,20 @@
 
 @AllArgsConstructor
 @Getter
-public class PackageArtifactCredentials implements ArtifactCredentials {
-  private final String name = "package";
+public class CloudFoundryArtifactCredentials implements ArtifactCredentials {
+  public static final String TYPE = "cloudfoundry/app";
+
+  private final String name = "cloudfoundry";
```

Comment:
CloudFoundryCloudProvider.ID is commonly used for this string

Reply:
Thanks. Eventually this string will be per-account when we expand `CloudFoundryArtifactCredentials` to be an artifact account you can select on stage "Produces Artifacts". It could be `cheeseburgers` right now and the effect would be nil.